README.md

```diff
@@ -10,30 +10,22 @@ status: draft
 
 # Introduction
 
-In this tutorial, we are going to build a simple web application using [AdonisJS](https://adonisjs.com/) and integrate it with [Materialize](https://materialize.com).
+In this tutorial, we are going to build a web application using [AdonisJS](https://adonisjs.com/) and integrate it with [Materialize](https://materialize.com) to .
```

Comment:
You should make the intent of the tutorial clearer in this first sentence (i.e. add something after "to...").